FEXCore/Source/Interface/Config/Config.cpp

```diff
@@ -113,11 +111,11 @@ fextl::string GetApplicationConfig(const std::string_view Program, bool Global)
   return fextl::fmt::format("{}{}.json", ConfigFile, Program);
 }
 
-void SetConfig(FEXCore::Context::Context* CTX, ConfigOption Option, uint64_t Config) {}
```

Comment:
Are these SetConfig/GetConfig functions even used anywhere? Feels like it might have been a refactor oops to leave it in?